src/pytest/CMakeLists.txt

```diff
@@ -5,18 +5,18 @@ SET(baseCommand "${CMAKE_CURRENT_SOURCE_DIR}/runtests.sh;--module=$<TARGET_FILE:
 FOREACH(n ${PY_TEST_FILES})
     GET_FILENAME_COMPONENT(test_name ${n} NAME_WE)
     ADD_TEST(NAME "PY_${test_name}"
-        COMMAND ${baseCommand} "-t" "${n}"
+        COMMAND ${baseCommand} "-t" "${n}" "--module-args=GC_POLICY DEFAULT"
```

Comment:
We should rename the test name as well, calling the currently-default GC "INPROCESS" or  "SYNC"